### ISSUE TITLE
Looking for help in reproduce NSYNTH (2018)

### ISSUE THREAD

**Issue by OriYitzhaki** [magenta/magenta#1877]
Hey,
I'm trying to reproduce the NSYNTH results on my own, And I have some issues (which might not be related).
1.  I'm using GPU: (RTX 2080Ti) vmem:10G which feels like plenty of memory and yet I'm getting OOM error when calculating the gradients with the original setting (4 secs data and large network).
2. When running for 100000 training steps (with a batch of size  1) the network learns to produce only silent audio.

For the 1st issue, I've reduced the number of nested layers (from 10X3 to 5X3) and also reduced the audio duration from 4secs to 1.6secs (25600 instead of 64000).
But for the 2nd issue, I don't know how to solve it. the training loss seems to have reached a plateau (the loss value is 2~)
any help would be appreciated

BTW I'm not sure if it's matters but I'm working with TF2 and keras and the training dataset is "nsynth-test".
